packages/monitors/src/ContractMonitor.js

```diff
@@ -157,7 +167,7 @@ class ContractMonitor {
         ". tx: " +
         createEtherscanLinkMarkdown(event.transactionHash, this.empProps.networkId);
 
-      this.logger.info({
+      this.logger[this.logOverrides.newPositionCreated || "info"]({
```

Comment:
I totally agree with this, when this alert was originally designed we did not anticipate so many new sponsor events